docs/3-index-pages/index-as-table.md

```diff
@@ -55,8 +55,6 @@ index do
 end
 ```
 
-## Defining Actions
```

Comment:
That was removed when I ran rake yard, but I didn't change the source file in that location.  I can add it back, but it had already been removed from the source documentation.